lib/orphans.nix

```diff
@@ -0,0 +1,104 @@
+{ lib }:
+let
+  inherit (lib)
+    types
+    optional
+    isDerivation
+    attrByPath
+    flatten
+    any
+    assertMsg
+  ;
+  inherit (builtins)
+    length
+    concatStringsSep
+    mapAttrs
+    attrValues
+    tryEval
+  ;
+  isAttr = types.attrs.check;
+in
+rec {
+  /* Return a list of maintainers from a package, empty list if inaccessible.
+
+     Example:
+       listMaintainers pkgs.foo
+       => [ { ... } ]
```

Comment:
Can you expand the output of `{ ... }`. I know this is the way `nix repl` shows, but it would be interesting to see at least one entry expanded so we can see which fields this attrset possibly contains.